src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_base_layout.html.twig

```diff
@@ -143,11 +143,6 @@
     {%- endif -%}
 {%- endblock dateinterval_widget -%}
 
-{% block choice_widget_collapsed -%}
```

Comment:
shouldn't this be kept?

Reply:
Since Bootstrap 5, select no longer use the `.form-control` class for styling but `.form-select`. Keeping this block in the base layout would make select render with `form-control form-select` for the attribute class which lead to unwanted effects. I have duplicate this block in both `bootstrap_3_layout.html.twig` and `bootstrap_4_layout.html.twig` and change it in `bootstrap_5_layout.html.twig`